modular_skyrat/modules/modular_implants/code/nifsofts/scryer.dm

```diff
@@ -0,0 +1,105 @@
+/obj/item/disk/nifsoft_uploader/scryer
+	name = "NIFSoft Scryer Uploader Disk"
+	loaded_nifsoft = /datum/nifsoft/scryer
+
+/datum/nifsoft/scryer
+	name = "Scryer"
```

Comment:
NIFLink Holocaller

program_desc "This ubiquitous NIFSoft adds Scryer functionality similar to MODSuits to the user's NIF; allowing for real-time communication through AR hologlass screens from a hardlight projector sat around the wearer's neck."